### PR TITLE
[MINOR][DOC] Fix a few markdown typos

### PR DESCRIPTION
**Pull request by taoran92**
## What changes were proposed in this pull request?

Easy fix in the markdown.

## How was this patch tested?

jekyII build test manually.

Please review http://spark.apache.org/contributing.html before opening a pull request.

### CODE REVIEW

**Comment by felixcheung** [comment]
ok to test

**Comment by felixcheung** [comment]
jenkins, retest this please

File: docs/mllib-feature-extraction.md [hunk 1]
```diff
@@ -278,8 +278,8 @@ for details on the API.
 multiplication. In other words, it scales each column of the dataset by a scalar multiplier. This
 represents the [Hadamard product](https://en.wikipedia.org/wiki/Hadamard_product_%28matrices%29)
 between the input vector, `v` and transforming vector, `scalingVec`, to yield a result vector.
-Qu8T948*1#
```
Comment (HyukjinKwon) [inline docs/mllib-feature-extraction.md]: @jkbradley, would you mind if I ask this was mistakenly written for sure?

**Comment by Lemonjing** [comment]
@HyukjinKwon 

The description of ElementwiseProduct is obvious, and i think "`Qu8T948*1#`" is a mistake, if not, "`Qu8T948*1#`" is a input vector and 1 is scaling vector, and this is not a good example to express vector, even so, there is markdown syntax error. By the way, there is no example context here, such as "e.g."，"for example" or something else. 

And i have queried the history and found another evidence. Before spark v1.6.0, this file do not have "`Qu8T948*1#`". So this maybe a mistake in one commit at v1.6.0 and this mistake not be found or fixed across v1.6.0 to now.

...
https://github.com/apache/spark/blob/v1.4.1/docs/mllib-feature-extraction.md no
https://github.com/apache/spark/blob/v1.5.0/docs/mllib-feature-extraction.md no
https://github.com/apache/spark/blob/v1.5.2/docs/mllib-feature-extraction.md no
https://github.com/apache/spark/blob/v1.6.0/docs/mllib-feature-extraction.md yes
1.6.0 ~ 2.3.0 yes

Thanks, and hope to confirm it.

File: docs/mllib-feature-extraction.md (hunk 1 above)
Comment (Lemonjing) [reply docs/mllib-feature-extraction.md]: @HyukjinKwon  If the original author did not reply, you can look at my latest reply and confirm this confused character sequence.

Comment (HyukjinKwon) [reply docs/mllib-feature-extraction.md]: he is an active commiter / pmc. let's wait for few more days before we go ahead.

**Review by HyukjinKwon — APPROVED** [review]

**Comment by Lemonjing** [comment]
see http://spark.apache.org/docs/latest/ml-features.html#elementwiseproduct

File: docs/mllib-pmml-model-export.md [hunk 2]
```diff
@@ -7,15 +7,15 @@ displayTitle: PMML model export - RDD-based API
 * Table of contents
 {:toc}
 
-## `spark.mllib` supported models
```
Comment (Lemonjing) [inline docs/mllib-pmml-model-export.md]: backquotes in mds cause display problems (see http://spark.apache.org/docs/latest/mllib-pmml-model-export.html)

Comment (HyukjinKwon) [reply docs/mllib-pmml-model-export.md]: nice

**Comment by HyukjinKwon** [comment]
@felixcheung, does this look good to you too?

**Review by felixcheung — APPROVED** [review]

**Comment by HyukjinKwon** [comment]
Merged to master and branch-2.3.

**Pull request closed without merging**